MetaMask/metamask-extension
# Show product tour only on home page

Product Tour doesn't allow user to click anywhere till they finish the tour completely. But with the portfolio, users can go directly to swaps. This PR is to display relevant product tour items only on the home page.

* Fixes #19929 

## Screenshots

Should be the same, with no UI change


## Manual Testing Steps

- Go to popup view after finishing onboarding
- Check the product tour is only on the home page
- For engineers to review with code, change the `showProductTour` condition in the `app-header.js`

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
